Tiltfile

```diff
@@ -80,15 +81,21 @@ def build_crds():
 # deploy_gatekeeper defines the deploy process for the gatekeeper chart from manifest_staging/charts/gatekeeper.
 def deploy_gatekeeper():
     local("kubectl create namespace gatekeeper-system || true")
-
+    
+    local_resource(
```

Comment:
This preserves the types of all helm values rather than formatting them as strings.